src/segment-loader.js

```diff
@@ -1458,6 +1477,7 @@ export default class SegmentLoader extends videojs.EventTarget {
         video: true
       };
 
+      this.mediaConfig_ = trackInfo;
       this.startingMedia_ = trackInfo;
```

Comment:
I think it may be better to have more descriptive names here, since technically `startingMedia` is the `currentMediaConfig` and `mediaConfig` is the `startingMediaConfig`.

Reply:
I'm hesitant to make such a change, at least in this pull request as that will require a lot of code changes in many different locations, perhaps we can make such a change in another pull request.